src/main/java/nextstep/subway/application/LineService.java

```diff
@@ -59,18 +91,22 @@ public void updateLineById(Long id, LineRequest lineRequest) {
 
     @Transactional
     public void deleteLine(Long id) {
-        lineRepository.deleteById(id);
+        Line line = lineRepository.findById(id).orElseThrow(LineNotFoundException::new);
```

Comment:
습관적으로 위와 같이 컨벤션을 유지해주시면 좋을 것 같아요 :)

```suggestion
        Line line = lineRepository.findById(id)
                .orElseThrow(LineNotFoundException::new);
```